doc/source/skips/1-governance.rst

```diff
@@ -111,6 +111,8 @@ long-term, continued commitment to the project and its :ref:`values`. A
 nomination will result in discussion that cannot take more than a month and
 then admission to the SC by consensus.
 
+The scikit-image steering council may be contacted at
+[skimage-steering@groups.io](mailto:skimage-steering@groups.io).
```

Comment:
This is markdown in a ReST document.

Reply:
:rofl:

Reply:
on a related note, did you know that one can build sphinx doc from a mixture of `rest` and `md` files? I find out this recently, including cross-links to pages. ReST is superior IMO for references (unfortunately no `:func:` in Markdown :-() but if we need to include a Markdown file at some point this is possible (with the `recommonmark` extension)

Reply:
@emmanuelle very interesting! Do you have a link to an example configuration/implementation?